cli/args/mod.rs

```diff
@@ -1750,28 +1701,16 @@ impl CliOptions {
         .as_ref()
         .map(|s| matches!(s.kind, NpmProcessStateKind::Byonm))
         .unwrap_or(false)
-      || self
-        .maybe_config_file()
-        .as_ref()
-        .map(|c| c.has_unstable("byonm"))
-        .unwrap_or(false)
+      || self.workspace.has_unstable("byonm")
   }
 
   pub fn unstable_sloppy_imports(&self) -> bool {
     self.flags.unstable_config.sloppy_imports
-      || self
-        .maybe_config_file()
-        .as_ref()
-        .map(|c| c.has_unstable("sloppy-imports"))
-        .unwrap_or(false)
+      || self.workspace.has_unstable("sloppy-imports")
```

Comment:
I must have missed it, but how are unstable features enabled in the workspace setting? I hope it's only at the top level?

Reply:
Yeah, it's only at the top level. Otherwise it surfaces diagnostics in deno_config.